app/controllers/admin/events/list.js

```diff
@@ -75,11 +75,6 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
          cellComponent : 'ui-table/cell/cell-speakers-dashboard',
          width         : 90
        },
-       {
```

Comment:
@iamareebjamal I think we should comment it out, As `ticket statistics` have played a crucial role in our tables. We will need it once the DOS issues are resolved?

Reply:
This is a git repository, not a college project in a pen drive that something which is needed in the future needs to be commented out instead of deleting. It has the entire history of changes in the repo, any deleted thing can be recovered.

> We will need it once the DOS issues are resolved

Then it'll add the DOS issue again. When is it a good idea to fetch unbounded one to many relationships of a list of objects? Answer - Never
If it'll need to be added, it'll be completely reimplemented